scrapers/freshmaker.py

```diff
@@ -0,0 +1,84 @@
+# SPDX-License-Identifier: GPL-3.0+
+from __future__ import unicode_literals
+import urllib3
+
+from scrapers.base import BaseScraper
+from purview.models.freshmaker import FreshmakerEvent, ContainerBuilds
+from scrapers.utils import retry_session
+from purview.utils.general import timestamp_to_datetime
+from purview import log
+
+urllib3.disable_warnings()
+
+
+class FreshmakerScraper(BaseScraper):
+
+    def run(self, since=None):
+        """
+        Main function that runs the Freshmaker scraper
+        :param since: a string representing a datetime to start scraping data from
+        :return: None
+        """
+        if since:
+            log.info('Start date: {0} does not apply to Freshmaker scraper!'.format(since))
```

Comment:
How about a message like `Ignoring the since parameter since it doesn't apply to the Freshmaker scraper.`